languages.yaml

```diff
@@ -1788,6 +1788,39 @@ Odin:
       - "#4797F3"
       - "#499AF7"
     chip: "#60AFFE"
+OpenScad:
+  type: programming
+  ascii: |
+    {0}                 ________
+    {0}             .:------------:   ...
+    {0}          .-+=             =+=-    :.
+    {0}        :+***               +**=:   .:.
+    {0}       =*****+             +*****=    -.
+    {0}      +**********+++++++**********+    -
+    {0}     =*****************************=   -
+    {0}    .*+==+********************+++***. .:
+    {0} .::-:.   .-+***************-     =*-..
+    {0}-.    .:.    -************=        *.
+    {0}=       .:.   :**********-        :+
+    {0}=         :.   =********=        .+.
+    {0}.-         -   .********-       :=.
+    {0} .-.       :   =********+.   .:=-
+    {0}   ::     .-:=************++==:
+    {0}     ·...·   .-=++*****++=-.
+    {0}                ---------
+  colors:
+    ansi: # required, a list of the ANSI colors used to colorize the logo
+      - blue
+      - magenta
+      - magenta
+      - white
+    hex: # optional, alternative to basic colors for terminals that support true colour.
+      - "#9B4F97"
+      - "#67217A"
+      - "#803788"
+      - "#FFFFFF"
```

Comment:
You're declaring 4 colors but only using one.

More info on how to color an ascii logo: https://github.com/o2sh/onefetch/wiki/ascii-art

Reply:
fixed. Removed unnecessary color definitions.

Reply:
[94b5656](https://github.com/o2sh/onefetch/pull/1502/commits/94b565681ce3d352880d7b4b56ee1cf857bb5cf2) I added a second color with hex values to get it closer to the original logo https://vercel-f4f8n13fs-o2shs-projects.vercel.app/#OpenScad